sql/derived_tables/req_stats.sql

```diff
@@ -14,6 +15,14 @@ FROM
     reshare_rs.patron_request_audit pra
     LEFT JOIN reshare_rs.status s ON pra.pra_to_status_fk = s.st_id
     LEFT JOIN reshare_rs.status s2 ON pra.pra_from_status_fk = s2.st_id
+    JOIN (
+        SELECT
+            *
+        FROM
+            reshare_rs.directory_entry de
+        WHERE
+            de.de_parent IS NULL
+            AND de.de_status_fk IS NOT NULL) AS names ON pra."__origin" = names."__origin"
```

Comment:
Quotation marks should not be needed here.

If they are being added by the SQL editor, it might be possible to configure the editor not to add them.